spiceaidocs/docs/components/data-connectors/clickhouse.md

```diff
@@ -4,36 +4,64 @@ sidebar_label: 'Clickhouse Data Connector'
 description: 'Clickhouse Data Connector Documentation'
 ---
 
-## Federated SQL query
-
-To connect to any Clickhouse database as connector for federated SQL query, specify `clickhouse` as the selector in the `from` value for the dataset.
+ClickHouse is a fast, open-source columnar database management system designed for online analytical processing (OLAP) and real-time analytics. This connector enables federated/accelerated SQL queries on top of a Clickhouse server.
 
 ```yaml
 datasets:
-  - from: clickhouse:path.to.my_dataset
+  - from: clickhouse:my.dataset
     name: my_dataset
 ```
 
 ## Configuration
 
+### `from`
+
+The `from` field for the Clickhouse connector takes the form of `from:path_to_my_dataset` where `path_to_my_dataset` is the path to the Dataset within Clickhouse. In the example above it would be `my.dataset`.
```

Comment:
We should also say that the first part before the period is the clickhouse DB (defaults to `clickhouse_db` parameter, or `default` if not specified)

Reply:
Fixed